tools/Calculation.py

```diff
@@ -113,9 +113,10 @@ def calculate_lower_and_upper_PDFuncertainty(central_measurement, pdf_uncertaint
     positive = []
     
     # split PDF uncertainties into downwards (negative) and upwards (positive) components
-    for index in range(1, 45):
+    for index in range(0, 100):
         pdf_weight = 'PDFWeights_%d' % index
         pdf_uncertainty = pdf_uncertainty_values[pdf_weight]
+        print index,central_measurement, pdf_uncertainty
         if index % 2 == 0:  # even == negative
```

Comment:
Is this still correct? The list used to start with an odd weight (positive) followed by the negative one.
The way it is currently done, the first weight will be counted as negative.
Could you please verify?